daprdocs/content/en/reference/components-reference/supported-middleware/middleware-wasm.md

```diff
@@ -0,0 +1,59 @@
+---
+type: docs
+title: "Wasm middleware"
+linkTitle: "Wasm"
+description: "Test your HTTP pipeline is functioning with the wasm middleware"
+aliases:
+- /developing-applications/middleware/supported-middleware/middleware-wasm/
+---
+
+The wasm [HTTP middleware]({{< ref middleware.md >}}) use a WASM file to handle request and response.
+
+## Component format
+
+In the following definition, it make content of request body into uppercase:
+
+```yaml
+apiVersion: dapr.io/v1alpha1
+kind: Component
+metadata:
+  name: wasm
+spec:
+  type: middleware.http.wasm.basic
+  version: v1
+  metadata:
+  - name: path
+    value: "./hello.wasm"
+  - name: runtime
+    value: "wazero"
+```
+
+## Spec metadata fields
+
+| Field | Details | Example |
+|-------|---------|---------|
+| path | The path of your WASM binary | "./hello.wasm" |
```

Comment:
```suggestion
| path | The path of your Wasm binary | "./hello.wasm" |
```

Reply:
It is relative to the config file, I will update it as absolute to clear.